.eslintrc

```diff
@@ -0,0 +1,33 @@
+---
+
+root: true
+
+env:
+  browser: true
+  mocha: true
+  node: true
+
+rules:
+  comma-dangle: 0
```

Comment:
Yeah, @Xe, some are using that horrifying syntax indeed.
Thankfully, as all fashions kid like to wear, it is losing its popularity :-)
Let's not do that, and anyway the current codebase is not using crazyness like this, so even better.

---

"Closing" (virtually) this line of comments as decision was reached: we keep `comma-dangle: 0` for the moment.